menuinst/api.py

```diff
@@ -89,11 +91,11 @@ def remove(
 @elevate_as_needed
 def install_all(
     *,
-    target_prefix: Optional[os.PathLike] = None,
-    base_prefix: Optional[os.PathLike] = None,
-    filter: Union[Callable, None] = None,
+    target_prefix: str | None = None,
+    base_prefix: str | None = None,
+    filter: Callable | None = None,
     _mode: _UserOrSystem = "user",
-) -> List[List[os.PathLike]]:
+) -> list[list[os.PathLike]]:
```

Comment:
```suggestion
) -> list[tuple[os.PathLike]]:
```

`install_all` calls `install`, which creates a list based on `Menu.create()` or `MenuItem.create()`. `create` returns `tuple`, not `list`.